lib/execution_engine2/sdk/SDKMethodRunner.py

```diff
@@ -319,6 +317,41 @@ def get_job_status_field(self, job_id, as_admin=False):
         """ Authorization Required: Read """
         return self.get_jobs_status().get_job_status(job_id=job_id, as_admin=as_admin)
 
+    def check_job_batch(
+        self,
+        parent_job_id,
+        check_permission=True,
```

Comment:
Does `check_job_batch` get called elsewhere, other than in Impl file? The implementation file doesn't have any way of adding the `check_permission` param.

Reply:
No. This is simply a copy of the way things are done for "check_job". We want the path from the IMPL to always check permissions. Do you want me to move it into the function definition instead?

Reply:
Easiest would probably be to add the param to the function call in the Impl file.

Reply:
@ialarmedalien We don't want the user to be able to bypass permissions, but we've used it in the past before we had the permissions request mocking and before we had the permissions cache.